en/08_Changelogs/6.0.0.md

```diff
@@ -41,6 +44,58 @@ title: 6.0.0 (unreleased)
   - [`getSchemaDataDefaults()` now includes attributes](#formfield-schema-data)
 - [Full list of removed and changed API (by module, alphabetically)](#api-removed-and-changed)
 
+## Change to commercially supported modules {#changes-to-support}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 6 lifecycle.
+
+Review the list of [Commercially Supported Modules for Silverstripe CMS 6](https://docs.silverstripe.org/en/6/project_governance/supported_modules/).
+
+### Modules loosing commercial support {#modules-loosing-support}
+
+With the release of Silverstripe CMS 6.0.0, the following modules will no longer be commercially supported:
```

Comment:
```suggestion
Some modules that were commercially supported in Silverstripe CMS 5 are not supported in Silverstripe CMS 6. Some of those modules provide CMS6-compatible versions. Others have been dropped altogether.

Just because a module is not "commercially supported", doesn't mean that you shouldn't be using it. Community supported modules are maintained on a "best-effort" basis. You should take this into consideration when choosing to install a community supported module in your project.

Email [community@silverstripe.org](mailto:community@silverstripe.org) if you are keen to maintain some of the modules that are no longer commercially supported.
```
Using the language from the CMS 5 changelog which has:
1. Explanation of what dropping support actually means
1. Call for maintainers for modules not commercially supported going forward

Reply:
Updated